packages/api-server/src/methods/gw-error.ts

```diff
@@ -71,7 +71,8 @@ const revertErrorMapping: RevertErrorMapping = {
   },
   // Error(string)
   "0x08c379a0": {
-    message: (args: any[]) => `Error(${args[0]})`,
+    message: (args: any[]) =>
+      `Error: VM Exception while processing transaction: reverted with reason string '${args[0]}'`,
```

Comment:
seems godwoken-tests needs to update with the new format https://github.com/nervosnetwork/godwoken-web3/runs/7298443991?check_suite_focus=true#step:20:904

Reply:
Updated: https://github.com/nervosnetwork/godwoken-tests/pull/147